lib/components/rails/caching.rb

```diff
@@ -34,6 +34,18 @@ def cache_fragment_name
       def virtual_path
         "#{component_path}/#{action_name}"
       end
+
+      def instrument_payload(key)
```

Comment:
@joeosburn-desiringgod where do you think we are triggering code that uses this?

Reply:
@benhutton https://github.com/rails/rails/blob/b5db73076914e7103466bd76bec785cbcfe88875/actionpack/lib/abstract_controller/caching/fragments.rb#L161-L163